torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -463,6 +463,7 @@ struct Environment {
                "__round__",
                std::make_shared<BuiltinFunction>(aten::round, at::nullopt))},
           {"hash", std::make_shared<BuiltinFunction>(aten::hash, at::nullopt)},
+          {"id", std::make_shared<BuiltinFunction>(prim::id, at::nullopt)},
```

Comment:
It should still have an entry in `_builtins.py` (in addition to this one) just in case it gets shadowed e.g.

```python
my_id = id

@torch.jit.script
def fn(x):
	return my_id(x)
```

Reply:
i would prefer if the builtins are treated consistently instead of this being the only one to work that way. also, it's a very unlikely use case.